scopes/react/react/webpack/html.ts

```diff
@@ -12,11 +12,13 @@ export function html(title: string) {
       try { window.__REACT_DEVTOOLS_GLOBAL_HOOK__ = window.parent.__REACT_DEVTOOLS_GLOBAL_HOOK__; } catch {}
       </script>
       <!-- minimal css resets -->
-      <style> html { height: 100%; } body { margin: 0; height: 100%; } #root { height: 100%; } </style>
+      <style> body { margin: 0; } </style>
```

Comment:
also a bit risky and might break a few things

Reply:
if I remember correctly, this is also used for the fallback message when no preview. we need to test it there as well